sites/feedback/ajax/page-feedback-en.html

```diff
@@ -0,0 +1,84 @@
+<div id="gc-pft" class="row wb-disable-allow" data-wb-jsonmanager='{
+	"name": "gc-pft",
+	"extractor": [
+		{ "selector": "title", "path": "pageData/pageTitle" },
+		{ "selector": "html", "attr": "lang", "path": "pageData/language" },
+		{ "interface": "locationHref", "path": "pageData/submissionPage" },
+		{ "selector": "#wb-lng ul li:first-child a[lang]", "attr": "href", "path": "pageData/oppositelang" },
+		{ "selector": "[data-feedback-theme]", "attr": "data-feedback-theme", "path": "pageData/themeopt" },
+		{ "selector": "[data-feedback-section]", "attr": "data-feedback-section", "path": "pageData/sectionopt" },
+		{ "selector": "meta[name=\"dcterms.creator\"]", "attr": "content", "path": "pageData/institutionopt" },
+		{ "selector": "[data-feedback-link]", "attr": "data-feedback-link", "path": "contact/link" },
+		{ "selector": "[data-feedback-url]", "attr": "data-feedback-url", "path": "contact/url" }
+	]
+}'>
+	<div class="col-sm-10 col-md-9 col-lg-8">
+		<section class="well mrgn-bttm-0">
+			<h3 class="wb-inv">Give feedback about this page</h3>
+			<form action="https://feedback-retroaction.canada.ca" method="post" class="wb-postback wb-disable-allow" data-wb-postback='{"success":".gc-pft-thnk"}'>
```

Comment:
```suggestion
			<form action="https://feedback-retroaction.canada.ca/api/QueueProblemForm" method="post" class="wb-postback wb-disable-allow" data-wb-postback='{"success":".gc-pft-thnk"}'>
```

My understanding is they are going to keep the same path and query for the URL. Only the domain name are going to change.